sklearn/metrics/ranking.py

```diff
@@ -149,14 +141,34 @@ def average_precision_score(y_true, y_score, average="macro",
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
+    interpolation : None (default), or string 'eleven_point'
```

Comment:
Done. Although since we only have one option for `interpolation`, would this be better as a bool? So `interpolation=False` is the default and corresponds to the `None` option here, and `interpolation=True` corresponds to `interpolation='eleven_point'`. We could even change the parameter name to `eleven_point_interpolate` to make it extra clear. I have no particular preference, except that I could imagine people having to repeatedly check the docstring to remember that they should be typing `'eleven_point'`.